openedx/core/djangoapps/credentials/utils.py

```diff
@@ -42,7 +43,7 @@ def get_credentials_api_client(user):
     Arguments:
         user (User): The user to authenticate as when requesting credentials.
     """
-    scopes = ['email', 'profile', 'user_id']
+    scopes = ["email", "profile", "user_id"]
```

Comment:
I spent a little fair amount of time fighting with formatter settings that would make the linter happy, because these files have some pretty old code.  As a side effect I  ended up moving to these more modern standards, so there's a lot of these style changes. I apologise that it makes the PR harder to read. Some of these files haven't been touched in a couple of years and don't pass our current linter.

Reply:
Ah, yes, I've had this bite me before too. Maintenance fun! 🎉 🌮

Reply:
Thanks for updating all of these pesky style changes!